### PR TITLE
Update Observe pipeline output

### PR DESCRIPTION
**Pull request by joey-skittlz**
Signed-off-by: Joe Welsh <joe@observeinc.com>

### CODE REVIEW

File: pipeline/outputs/observe.md [hunk 1]
```diff
@@ -10,13 +10,12 @@ The following are the specific HTTP parameters to employ:
...
-| host                       | IP address or hostname of Observe's data collection endpoint                                                                                                                                                                                                                                                                       | collect.observeinc.com |
+| host                       | IP address or hostname of Observe's data collection endpoint.  $(OBSERVE_CUSTOMER) is your [Customer ID](https://docs.observeinc.com/en/latest/content/common-topics/HelpfulHints.html?highlight=customer%20id#customer-id)                                                                                                                                                                                                                                                                      | ${OBSERVE_CUSTOMER}.collect.observeinc.com |
```
Comment (patrick-stephens) [inline pipeline/outputs/observe.md:13]: So does this require an environment variable to be set for the default? I can imagine that might not be desirable in containers.

Comment (joey-skittlz) [reply pipeline/outputs/observe.md:13]: Actually, it's meant to represent a token of sorts, to replace OBSERVER_CUSTOMER with a customer's Observe Customer ID when host is added to OUTPUTS.  Sorry for the confusion, does this make sense?

i.e. if my Observe customer id was "joe" then my host would now be https://joe.collect.observeinc.com

Comment (patrick-stephens) [reply pipeline/outputs/observe.md:13]: Looks like that is what Grafana Cloud docs say: https://docs.fluentbit.io/manual/pipeline/outputs/loki#fluent-bit-+-grafana-cloud
Unfortunately the others seem a little more like real examples: https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch#aws-es

Comment (joey-skittlz) [reply pipeline/outputs/observe.md:13]: @patrick-stephens yes, it's just a KEY.  I tried to make it obvious (and consistent with Observe documentation) by using OBSERVE_CUSTOMER.    If I updated the sample [OUTPUT] example to say:

host         my-observe-customer-id.collect.observeinc.com

Would this resolve this?

Comment (joey-skittlz) [reply pipeline/outputs/observe.md:13]: Thanks @patrick-stephens !

**Comment by patrick-stephens** [comment]
@joey-skittlz ping me if you want me to review any updates

**Comment by joey-skittlz** [comment]
@patrick-stephens I assumed I made all necessary updates, have been waiting on you to commit.  Is there something I missed?

File: pipeline/outputs/observe.md (hunk 1 above)
Comment (patrick-stephens) [reply pipeline/outputs/observe.md:13]: Can you remove the `${...}` syntax as that is the exact syntax used for variables?

**Comment by joey-skittlz** [comment]
@patrick-stephens sorry about that, I just removed them.  The example provided should seem familiar to Observe customers as well.  Thanks for understanding.  Please let me know if you need anything else.

File: pipeline/outputs/observe.md [hunk 2]
```diff
@@ -29,18 +28,18 @@ In your main configuration file, append the following _Input_ & _Output_ section
...
-    http_passwd  ${OBSERVE_TOKEN}
     uri          /v1/http/fluentbit
 
     format       msgpack
+    header       Authorization     Bearer ${OBSERVE_TOKEN}
     header       X-Observe-Decoder fluent
     compress     gzip
+
+    # For Windows: provide path to root cert
+    #tls.ca_file  C:\td-agent-bit\isrgrootx1.pem
```
Comment (patrick-stephens) [inline pipeline/outputs/observe.md:43]: Ah, not sure if this was just added but `td-agent-bit` is deprecated and removed from 1.9.9 so could you switch it to `fluent-bit`?

Comment (joey-skittlz) [reply pipeline/outputs/observe.md]: Done.

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**